File: HackerRank/30-Days-of-Code/C++/day07-Arrays.cpp
#include <iostream>
#include <string>
using namespace std;

//Question Link : https://www.hackerrank.com/challenges/30-arrays/problem?isFullScreen=true

int main()
{
    int l;
    cin >> l;
    int* N=new int[l];
    
        for(int i=0;i<l;i++){
            cin>>N[i];
        }
        
        for(int j=l-1;j>=0;j--){
           cout << N[j] << " ";
        }
    
    delete[] N;

    return 0;
}

File: HackerRank/30-Days-of-Code/Java/day07Arrays.java
import java.util.Scanner;

//Question Link : https://www.hackerrank.com/challenges/30-arrays/problem?isFullScreen=true


public class day07Arrays {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        
        int l=scan.nextInt();
        scan.nextLine();
        int[] N=new int[l];
        
        for(int i=0;i<l;i++){
            N[i]=scan.nextInt();
        }
        
        for(int j=l-1;j>=0;j--){
            System.out.print(N[j] + " ");
        }
        
        scan.close();
    }
}
